src/main/java/in/projecteka/consentmanager/user/SignUpRequestValidator.java

```diff
@@ -168,7 +168,7 @@ private static boolean isUnique(List<Identifier> unverifiedIdentifiers,
 
     private static Validation<String, String> validatePassword(String password) {
         if (Strings.isNullOrEmpty(password)) {
-            return Validation.invalid("password can't be empty");
+            return Validation.valid(password);
         }
```

Comment:
This does not need to be changed now right.

Reply:
We can do this as part of separate story. Your story is about letting the user login OTP.